cicd/Jenkins/Jenkinsfile

```diff
@@ -681,6 +690,29 @@ pipeline {
           }
         }
       }
+
+      stage('Create PR') {
+        agent {
+            dockerfile {
+              filename 'Dockerfile'
+              dir 'cicd/docker/autopr'
+              label 'Docker-enabled'
+            }
+        }
+        steps {
+          script {
+            try {
+              if ( BRANCH_NAME == PROD_BRANCH ) {
+                createPRInShellVueChi()
+                createPRInShellChi()
+              }
+            } catch (Exception e) {
+              echo "The creation of PRs in the shell failed"
```

Comment:
Include the output of the exception in the log for troubleshooting.

Reply:
I've added here the error message with `${e.getMessage()}`